client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/domain-only.tsx

```diff
@@ -0,0 +1,88 @@
+import { Button } from '@wordpress/components';
+import { translate } from 'i18n-calypso';
+import emailImage from 'calypso/assets/images/thank-you-upsell/email.svg';
+import ThankYouV2 from 'calypso/components/thank-you-v2';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { emailManagement } from 'calypso/my-sites/email/paths';
+import { useSelector } from 'calypso/state';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { getDomainPurchaseType } from '../../utils';
+import ProductDomain from '../products/product-domain';
+import type { ReceiptPurchase } from 'calypso/state/receipts/types';
+
+interface DomainOnlyThankYouContainerProps {
+	purchases: ReceiptPurchase[];
+}
+
+export const getDomainFooterDetails = ( limit?: number ) => {
```

Comment:
Why do we export that function? It's only used in this file.

Reply:
The same details may be used in other pages.

For example, the Domain Transfer to User congrats page: https://github.com/Automattic/wp-calypso/pull/86201/files#diff-62bfed7fed3f6cf08bb8514d04023c74377969d1dab7104bb6670f0937386386R55 

As well as the Generic Page that shows multiple products: https://github.com/Automattic/wp-calypso/pull/86086/files#diff-0d0bc3b66f1e56ca70d3c9bfaf1eb9c46f31e96f47f5dc1a915aafe3175b363bR39-R41

Reply:
I see. Then maybe we could move that function to a dedicated file? It feels a bit strange to import it from this DomainOnly page component.

Reply:
Yeah, it does feel a bit weird to have that function in the page component. I was sort of on the fence about it, since I wanted to be able to update all relevant page content within one single file. However, I guess we would have to update the content in two different files anyway for other pages that share the same content as the Domain Only congrats page.

I'll move the function its own file!